shadows/framework/src/main/java/org/robolectric/shadows/ShadowContentResolver.java

```diff
@@ -530,6 +531,20 @@ protected static boolean isSyncActive(Account account, String authority) {
     return status != null && status.syncRequests > 0;
   }
 
+  @Implementation
+  protected static List<SyncInfo> getCurrentSyncs() {
+    List<SyncInfo> list = new ArrayList<>();
+    for (Map.Entry<String, Map<Account, Status>> map : syncableAccounts.entrySet()) {
+      for (Map.Entry<Account, Status> mp : map.getValue().entrySet()) {
```

Comment:
Is `map.getValue()` non-null?

Reply:
Here, `map.getValue()` represents the Map<Account, Status> map.
So, if map is null then map.getValue() can be null.
We can handle this case as: `if (map.getValue() == null) return null;`

Reply:
I think you should write it:
```
if (map.getValue() == null) {
    continue;
}
```